client_handler.cpp

```diff
@@ -1730,6 +1738,47 @@ bool ClientHandler::OnCertificateError(CefRefPtr<CefBrowser> browser,
 	return TRUE;
 }
 
+// Usually IsWindowsXXOrGreater version helper API should be used,
+// but there is no IsWindows11OrGreater API yet. 
+// NOTE: GetVersionEx return information which depends on manifest,
+// so use RtlGetVersion API. 
+bool ClientHandler::GetIsWindows11OrLater(HWND hWindow)

```

Comment:
Just a question, can't we use [IsWindowsVersionOrGreater](https://learn.microsoft.com/ja-jp/windows/win32/api/versionhelpers/nf-versionhelpers-iswindowsversionorgreater) ?
Can't we detect version by simply excuting `IsWindowsVersionOrGreater(11, 0, 0)` ?

Reply:
Oh, I've overlooked it!
Thanks.

Reply:
Hmm, [IsWindowsVersionOrGreater](https://learn.microsoft.com/ja-jp/windows/win32/api/versionhelpers/nf-versionhelpers-iswindowsversionorgreater) seems not to work correctly on Windows 11, the version number is 10.0 even on Windows 11. 

https://learn.microsoft.com/en-us/windows/win32/sysinfo/operating-system-version

Reply:
Surely. it doesn't accept dwBuildNumber.

Reply:
How about move this function to the `SBUtil` class in `sbcommon.h`?
We may use this function in other files.

Reply:
It looks good. I've fixed it.